docs/modules/ROOT/pages/index.adoc

```diff
@@ -460,7 +471,7 @@ To generate the COSE signature for an *<<_identity_assertion,identity assertion>
 
 In each of the above sections, the following changes MUST be applied:
 
-* Any reference to the _claim_ MUST be replaced with the CBOR serialization of the `referenced_assertions` field from the *<<_identity_assertion,identity assertion>>.*
+* Any reference to the _claim_ MUST be replaced with the CBOR serialization of the `tbs` field from the *<<_identity_assertion,identity assertion>>.*
```

Comment:
As noted previously, are there any requirements on the CBOR serialization?  Are they the same as used by C2PA - ["Core Deterministic Encoding Requirements" of CBOR](https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding)